apiserver/facades/client/spaces/spaces_rename.go

```diff
@@ -0,0 +1,241 @@
+/*
+ * // Copyright 2020 Canonical Ltd.
+ * // Licensed under the AGPLv3, see LICENCE file for details.
+ */
```

Comment:
Remove multi-line comment. I'm surprised this passes static analysis.

Reply:
dammn my copyright generator sucks. Will do